doc/src/manual/constructors.md

```diff
@@ -539,11 +549,11 @@ that specific type parameters cannot be requested manually.
 For example, say we define a type that stores a vector along with an accurate representation of
 its sum:
 
-```julia
-type SummedArray{T<:Number,S<:Number}
-    data::Vector{T}
-    sum::S
-end
+```jldoctest
```

Comment:
I guess making this a doctest at least verifies that the syntax doesn't error (or warn if people look closely enough), but otherwise it doesn't run anything if it's just a type declaration?